packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/services.ts

```diff
@@ -45,6 +45,10 @@ const computeSteps = ({
       return false
     }
 
+    // if(((step === STEPS.additionalInfo || step === STEPS.submitted ) && TIERS[2] === 2) && addExtraStep) {
```

Comment:
is better to not leave commented code

Reply:
off, yes, I forgot to remove that. thanks Juli!